parsec/core/gui/instance_widget.py

```diff
@@ -212,13 +212,20 @@ def login_with_password(self, key_file, password):
 
     def show_central_widget(self):
         self.clear_widgets()
-        central_widget = CentralWidget(
-            self.core,
-            self.core_jobs_ctx,
-            self.core.event_bus,
-            systray_notification=self.systray_notification,
-            parent=self,
-        )
+        try:
+            central_widget = CentralWidget(
+                self.core,
+                self.core_jobs_ctx,
+                self.core.event_bus,
+                systray_notification=self.systray_notification,
+                parent=self,
+            )
+        except AttributeError:
```

Comment:
It would be better to only try/except the access to the event_bus:
```python
try:
    core_event_bus = self.core.event_bus
except AttributeError:
    [...]
```

Reply:
I guess, we shall also do some sanity check with `core_jobs_ctx` to be sure that the widget is not built with None object.